dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/cache/DefaultCacheProvider.java

```diff
@@ -123,13 +123,33 @@ private long orZeroInTestRun( long value )
         return cache;
     }
 
+    private long getSize1()
```

Comment:
Maybe just have one method `size( long )` and access with `size( SIZE_1K )` to avoid having to add more methods later?